reproducers/openmp-target/reproducer-openmptarget-issue3.cpp

```diff
@@ -0,0 +1,23 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-21, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
```

Comment:
@MrBurmark we should pass this along to John as a standalone reproducer. Can you put a compile line in a comment in the file for John to try. Since the bug shows up with clang10 and clang10-gcc-8, you should include compilation instructions for both compilers.

Reply:
If you could, and that happens, CC me. This looks like something I want to get a handle on, because I have a feeling it's related to the new header handling stuff.

Reply:
@trws is John the right person to send this to or Chris? Or both?

Reply:
Probably both.  It's a clang bug so Chris should see it, but that also means it might show up in either CORAL compiler frontend too, so John might want to add it to what we should look out for from those compilers too.

Reply:
@MrBurmark do you have time to put together a single file standalone reproducer for this and send to John and Chris< cc'ing Tom and me?

Reply:
Ya, I just got caught up trying to figure out some of the performance stuff in the tiling with teams PR.